src/bridge/IrcHandler.ts

```diff
@@ -167,7 +169,6 @@ export class IrcHandler {
                         visibility: "private",
                         // We deliberately set our own power levels below.
                         // preset: "trusted_private_chat",
-                        invite: [toUserId],
```

Comment:
Hold on a sec, are we actually inviting the user at all now?

Reply:
`ensureMatrixUserJoined` should join the user right after the creation.

https://github.com/matrix-org/matrix-appservice-irc/blob/7dc74b92cd6906619e53c262ea14682fd901fb8f/src/bridge/IrcHandler.ts#L308-L311

Reply:
Works for me.
## Test plan
1. Rebuild & Restart bridge with this branch
2. As an IRC user, message a Matrix user.
3. Use `/nick new-name` to a new username.
4. Message the Matrix user again.

https://user-images.githubusercontent.com/10872136/116164333-e260e300-a6f9-11eb-968f-4f6beea07970.mp4